Co2 sankey fix

kaskranenburgQ opened the pull request:
This PR aims to solve issue [#4203](https://github.com/quintel/etmodel/issues/4203) from ETModel.

The main issue lay in that the hydrogen co2 streams were not queried correctly due to name changes from nodes.

mabijkerk commented:
@kaskranenburgQ I think the agriculture CHPs may not be handled properly in this chart either. Could you look into this?

- agriculture_chp_engine_network_gas_dispatchable
- agriculture_chp_engine_network_gas_must_run
- agriculture_chp_wood_pellets
- agriculture_chp_engine_biogas
- agriculture_heat_backup_burner_network_gas

kaskranenburgQ commented:
I updated the branch and took in the points from @mabijkerk. 

> A smaller change however would be to add a similar structure for energy_power_mv_distribution_network_electricity as we have for energy_power_hv_network_electricity in the query.

This was a great way to get rid of some of the hardcoded nodes.

I implemented the agriculture CHP's by using the nodes that have  agriculture_heat hourly calculations.
